Automattic/regenerate-thumbnails
# Fix vulnerabilities

### Overview

Fix the vulnerabilities updating the packages `node-sass`, `webpack` and `webpack-cli` and recreating the `yarn.lock`.

Also, the build script was updated to set the `--mode=production` explicitly to prevent the webpack warning.

### Testing

1. Run the build.
2. Create a zip with the plugin.
3. Install the zip in a WordPress site.
4. Upload a photo to a post.
5. Go to `Settings → Media` and update the Thumbnail size.
6. Go to `Tools → Regenerate Thumbnails`.
7. Regenerate the Thumbnails and check if everything works properly.
